crates/e2e/src/test_node.rs

```diff
@@ -32,14 +32,14 @@ use subxt::{
 };
 
 /// Spawn a local substrate node for testing.
-pub struct TestNodeProcess<R: Config> {
-    proc: process::Child,
+pub struct TestNode<R: Config> {
```

Comment:
Renamed TestNodeProcess -> TestNode, since proc is now optional this seems like a better name